samples/bluetooth/broadcast_audio_sink/src/main.c

```diff
@@ -270,15 +273,15 @@ static int lc3_enable(struct broadcast_sink_stream *sink_stream)
 		return ret;
 	}
 
-	frames_per_sdu = bt_audio_codec_cfg_get_frame_blocks_per_sdu(sink_stream->stream.codec_cfg,
-								     true);
+	frames_per_sdu =
+		bt_audio_codec_cfg_get_frame_blocks_per_sdu(sink_stream->stream.codec_cfg, true);
 
 #if defined(CONFIG_USB_DEVICE_AUDIO)
 	sink_stream->lc3_decoder = lc3_setup_decoder(frame_duration_us, freq_hz, USB_SAMPLE_RATE,
 						     &sink_stream->lc3_decoder_mem);
 #else
-	sink_stream->lc3_decoder = lc3_setup_decoder(frame_duration_us, freq_hz, 0,
-						     &sink_stream->lc3_decoder_mem);
+			sink_stream->lc3_decoder = lc3_setup_decoder(frame_duration_us, freq_hz, 0,
```

Comment:
I think you added two extra tabs by mistake here, and on the line below?

Reply:
Will fix it